src/extension/background.re

```diff
@@ -72,6 +72,35 @@ Chrome.ContextMenus.create {
   "onclick": fun _ tab => toggleConversion tab##id
 };
 
+let convertExt ext => switch ext {
+  | "ml"  => "mli"
+  | "mli" => "ml"
+  | "re"  => "rei"
+  | "rei" => "re"
+  | ext => ext
+};
+let getNewURL url => 
```

Comment:
`getNewURL` is a bit vague for a function at toplevel. How about `swapExtension` instead (it doesn't really matter whether it's a url or file path here either)